src/inverseRobot.js

```diff
@@ -23,8 +23,21 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const valueObj = Object.values(robot);
+  const keyObj = Object.keys(robot);
+  const copy = {};
```

Comment:
from checklist:
3.[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)
maybe, repairRobot as opposite to inverseRobot would be better